scripts/release/packager/download-dependency-xcframeworks.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function step { >&2 echo -e "\033[1m\033[36m* $@\033[0m"; }
+function finish { >&2 echo -en "\033[0m"; }
+trap finish EXIT
+
+CORE_VERSION=${1}
+COMMON_VERSION=${2}
+TELEMETRY_VERSION=${3}
+ARTIFACTS_DIRECTORY=${4}
+
+step "Cleaning up dependencies directory"
+rm -rf ${ARTIFACTS_DIRECTORY}
+mkdir ${ARTIFACTS_DIRECTORY}
+
+cd ${ARTIFACTS_DIRECTORY}
+
+step "Download MapboxCommon.xcframework from https://api.mapbox.com/downloads/v2/mapbox-common/releases/ios/packages/${COMMON_VERSION}/MapboxCommon.zip"
+curl -n https://api.mapbox.com/downloads/v2/mapbox-common/releases/ios/packages/${COMMON_VERSION}/MapboxCommon.zip --output Common.zip
+
+step "Download MapboxMobileEvents.xcframework from https://api.mapbox.com/downloads/v2/mobile-events/releases/ios/packages/${TELEMETRY_VERSION}/MapboxMobileEvents.xcframework.zip"
+curl -n https://api.mapbox.com/downloads/v2/mobile-events/releases/ios/packages/${TELEMETRY_VERSION}/MapboxMobileEvents.xcframework.zip --output Telemetry.zip
+
+step "Download MapboxCoreMaps.xcframework from https://api.mapbox.com/downloads/v2/mobile-maps-core/releases/ios/packages/${CORE_VERSION}/MapboxCoreMaps.xcframework.zip"
+curl -n https://api.mapbox.com/downloads/v2/mobile-maps-core/releases/ios/packages/${CORE_VERSION}/MapboxCoreMaps.xcframework.zip --output Core.zip
```

Comment:
how do all these SDK Registry downloads work without tokens?

Reply:
The `-n` make `curl` reference your ~/.netrc